src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,28 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  error: ErrorTypes | null,
+};
+
+export const Notification: React.FC<Props> = ({ error }) => {
+  const [onButtonDelete, setOnButtonDelete] = useState(true);
```

Comment:
The variable name 'onButtonDelete' is not clear, consider renaming it to 'isNotificationVisible' for better clarity.